server/views/pages/overview/partials/overviewTab/overviewTabContentsV2.njk

```diff
@@ -30,8 +30,21 @@
                     <span class="govuk-body govuk-!-font-size-36 govuk-!-font-weight-bold" data-qa="in-progress-goals-count">{{ goalCounts.activeCount }}</span>
                     <span class="govuk-tag govuk-tag--blue govuk-!-margin-left-2 govuk-!-margin-bottom-4">In progress</span>
                   </div>
-                  <a class="govuk-link govuk-!-font-size-19 govuk-!-margin-right-4" href="/plan/{{ prisonerSummary.prisonNumber }}/view/goals#in-progress-goals" data-qa="view-in-progress-goals-button">View in progress goals</a>
+                  {% if featureToggles.completedGoalsEnabled %}
```

Comment:
Not sure I follow this - we have different markup for the In-Progress goals link based on whether the Completed Goals feature toggle is set? 🤔 
And the difference is that one has a right margin style and the other doesn't?

Are we sure about this? 🤔

Reply:
This was just to keep the layout consistent with what Karen asked for. But happy to look at a different approach.

![Selected October 17 2024 10:03:14](https://github.com/user-attachments/assets/217ceca0-b313-4ee5-b279-69fe6dfcae31)